http/server_test.ts

```diff
@@ -553,7 +553,7 @@ Deno.test({
       await assertThrowsAsync(
         () => badConn.read(new Uint8Array(1)),
         Deno.errors.InvalidData,
-        "invalid certificate: UnknownIssuer",
+        "invalid peer certificate contents: invalid peer certificate: UnknownIssuer",
```

Comment:
We should fix the error message to be less verbose in Deno. The old one seems to be on point.

Reply:
It will be hard as this error is surfaced from `rustls` and we'd have to manually rewrite it. I think it's not worth it.